cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -28,6 +28,7 @@
 from path import Path as path
 from six import text_type
 from waffle.testutils import override_switch
+from nose.plugins.attrib import attr
```

Comment:
Hm... not to be a pain, but when we sharded bok-choy we decided not to add more dependency on nose since we want to move off of it soon. Instead just adding the shard as a class attribute, like so: https://github.com/edx/edx-platform/blob/7820d2509c7bf7d61cde0ddf56620d3952c375f6/common/test/acceptance/tests/lms/test_unicode_username_admin.py#L15 . Might be a good idea to stick with that strategy, though I know you edited a lot of files.